data/brands/shop/jewelry.json

```diff
@@ -1224,6 +1224,17 @@
         "name:zh-Hant": "謝瑞麟",
         "shop": "jewelry"
       }
-    }
+    },
+	{
```

Comment:
Please fill this entry‘s Wikidata and displayName you want, sorry for haven't pointed it out in my first review. 😥

Reply:
No problem! I should have guessed that too must have a brand:wikidata. Adding it now!